project_admin/tests/tests_views.py

```diff
@@ -0,0 +1,221 @@
+        """
+        c = Client()
+        response = c.post("/project-admin/login/",
+                          {'password': 'test1234'},
+                          follow=True)
+        response = c.post("/project-admin/config-homepage-text/",
+                          {'faq': 'foo',
+                           'about': 'bar',
+                           'overview': 'foo-bar',
+                           'homepage_text': 'foobar!',
+                           'upload_description': 'foo_bar'},
+                          follow=True)
+        self.assertEqual(response.context['config'].faq, 'foo')
+        self.assertEqual(response.context['config'].about, 'bar')
+        self.assertEqual(response.context['config'].overview, 'foo-bar')
+        self.assertEqual(response.context['config'].homepage_text, 'foobar!')
+        self.assertEqual(response.context['config'].upload_description,
+                         'foo_bar')
+
+    def test_config_file_settings_logged_out(self):
+        """
+        Test config_file_settings function when logged out
+        """
+        c = Client()
+        response = c.get("/project-admin/config-file-settings/")
+        self.assertRedirects(response, '/project-admin/',
+                             status_code=302, target_status_code=302)
+
+    def test_post_config_file_settings(self):
+        """
+        Test making a post request to
+        config_file_settings.
+        """
+        c = Client()
+        response = c.post("/project-admin/login/",
+                          {'password': 'test1234'},
+                          follow=True)
+        response = c.post("/project-admin/config-file-settings/",
+                          {'file_description': 'foo',
+                           'file_tags': 'foo,bar', },
+                          follow=True)
+        self.assertEqual(response.context['config'].file_description, 'foo')
+        self.assertEqual(response.context['config'].file_tags,
+                         '["foo", "bar"]')
+
+    def test_get_config_file_settings(self):
+        """
+        Test making a get request to
+        config_file_settings.
+        """
+        c = Client()
+        response = c.post("/project-admin/login/",
+                          {'password': 'test1234'},
+                          follow=True)
+        response = c.get("/project-admin/config-file-settings/")
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response,
+                                'project_admin/config-file-settings.html')
+
+    def test_config_oh_settings_logged_out(self):
+        """
+        Test config_oh_settings function when logged out
+        """
+        c = Client()
+        response = c.get("/project-admin/config-oh-settings/")
+        self.assertRedirects(response, '/project-admin/',
+                             status_code=302, target_status_code=302)
+
+    def test_post_config_oh_settings(self):
```

Comment:
This test now fails because #45 changed how the upload works. The easiest way to fix this for now is to remove this test. Then we can merge #46 and can write a working test for this later.

Reply:
If this is the case, I think I'll have to remove the whole of `test_config_oh_settings_logged_out`

Reply:
Sorry, my bad. I commented on the wrong function. The problem is the `test_post_config_file_settings` function that doesn't work right now ([check the log at travis](https://travis-ci.org/gedankenstuecke/oh_data_uploader/jobs/349463887)). You could start by removing this test function for now and then see if travis passes then.